### PR TITLE
Added 2017 Week 2 announce

### PR DESCRIPTION
**Pull request by VoyTechnology**

### CODE REVIEW

File: source/_posts/2017-week-2-announce.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Week 2 | Welcom | EGM | Intro to Redbrick
```
Comment (GoldenBadger) [inline source/_posts/2017-week-2-announce.md]: Welcom*e*

File: source/_posts/2017-week-2-announce.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+date: 2017-09-26 22:07:50
+author: voy
+tags:
+  - EGM
+  - Freshers
+  - Ball
+---
+Dear Brickies, old, and new
```
Comment (GoldenBadger) [inline source/_posts/2017-week-2-announce.md]: Dear Brickies, old and new,

File: source/_posts/2017-week-2-announce.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+First of all, we would like to welcome all new members to Redbrick, the best
+society in DCU, holding strong since 1996.
+
+## Intro To Redbrick
+
+To get you started with, we have a n00b friendly introduction session into what
```
Comment (GoldenBadger) [inline source/_posts/2017-week-2-announce.md]: drop "with"

File: source/_posts/2017-week-2-announce.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+Redbrick does, why you should care, and how to connect to Redbrick to be part of the hipster IRC squad. The talk will be held on Wednesday, 2017-09-27 16:00 IST.  
+https://www.facebook.com/events/134674780502310
+
+## EGM
+
+We are also holding a EGM, as we need to elect a First Year Rep - a proxy between what the first years need and the rest of committee (because we can't understand kids those days). There will be pizza. The EGM would take place in Week 3, on Wed 2017-10-04 17:00
```
Comment (GoldenBadger) [inline source/_posts/2017-week-2-announce.md]: EGM *will* take place

File: source/_posts/2017-week-2-announce.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+## Renewals
+
+We would also like to remind existing members about importance of renewals and unit tests. You can either renew on the website or send the money by PayPal. Just make sure that you email admins@redbrick.dcu.ie with your transaction code so we can verify.
```
Comment (GoldenBadger) [inline source/_posts/2017-week-2-announce.md]: about *the* importance

File: source/_posts/2017-week-2-announce.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+
+## Engineering and Computing Ball
+
+The Second ever Engineering and Computing Ball with take place on November 16th in The Bonnington Hotel (formerly Regency Hotel). You can sign up today by throwing 10 euro at a committee member.
```
Comment (GoldenBadger) [inline source/_posts/2017-week-2-announce.md]: The *s*econd ever [...] *will* take place [...] in *t*he Bonnington

File: source/_posts/2017-week-2-announce.md (hunk 4 above)
Comment (CSmartt) [reply source/_posts/2017-week-2-announce.md]: _an_ EGM

Comment (butlerx) [inline source/_posts/2017-week-2-announce.md]: no fb event?

File: source/_posts/2017-week-2-announce.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+We would also like to remind existing members about importance of renewals and unit tests. You can either renew on the website or send the money by PayPal. Just make sure that you email admins@redbrick.dcu.ie with your transaction code so we can verify.
+
+
+## Engineering and Computing Ball
+
+The Second ever Engineering and Computing Ball with take place on November 16th in The Bonnington Hotel (formerly Regency Hotel). You can sign up today by throwing 10 euro at a committee member.  
+https://www.facebook.com/events/312772709198090
+
+
+Your favourite secretary,
```
Comment (butlerx) [inline source/_posts/2017-week-2-announce.md]: these lines will become one when rendered

File: source/_posts/2017-week-2-announce.md (hunk 4 above)
Comment (GoldenBadger) [reply source/_posts/2017-week-2-announce.md]: None currently exists at time of writing

File: source/_posts/2017-week-2-announce.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+## Engineering and Computing Ball
+
+The Second ever Engineering and Computing Ball with take place on November 16th
+in The Bonnington Hotel (formerly Regency Hotel). You can sign up today by
+throwing 10 euro at a committee member.  
+https://www.facebook.com/events/312772709198090
+
+
+Your favourite secretary,
+voy
```
Comment (butlerx) [inline source/_posts/2017-week-2-announce.md]: these 2 tines will become one you know same above with fb links

**Comment by butlerx** [comment]
closed in favour of #230

**Pull request closed without merging**